.idea/libraries/Maven__com_fasterxml_classmate_1_1_0.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
```

Comment:
Let's try to remove .idea and target from github.
(do this in a separate branch)
1. Create a separate folder for another copy of your project.
2. git clone into selected directory.
3. Without IDEA, delete .idea and target folders.
4. Commit and push changes.
5. Check that the folders dissappeared from github.
6. Merge to develop branch.